community-content/alphafold_on_workbench/Dockerfile

```diff
@@ -59,27 +41,33 @@ RUN conda update -qy conda \
       python=3.7
 
 COPY . /app/alphafold
-RUN wget -q -P /app/alphafold/alphafold/common/ \
-  https://git.scicore.unibas.ch/schwede/openstructure/-/raw/7102c63615b64735c4941278d92b554ec94415f8/modules/mol/alg/src/stereo_chemical_props.txt
 
 # Install pip packages.
 RUN pip3 install --upgrade pip \
     && pip3 install -r /app/alphafold/requirements.txt \
-    && pip3 install --upgrade jax jaxlib==0.1.69+cuda${CUDA/./} -f \
+    && pip3 install py3Dmol tqdm \
```

Comment:
These dependencies must have been installed elsewhere, but are needed for the notebook.